helm/flowforge/values.yaml

```diff
@@ -36,6 +36,13 @@ forge:
   support:
     enabled: false
 
+  rate_limits:
+    enabled: false
+    global: true
+    max: 1000
+    maxAnonymous: 10
```

Comment:
```suggestion
```

According to our defaults and docs, this should not be set.

Reply:
perhaps it could be `undefined` if it needs to be present?

Reply:
Having is as an example value in the flowforge.yml **STRONGLY** implies it is the default value. If that is not the case then it needs be updated.

Also it's not clear, is this a fastify-rate-limit value or something you've added?